### PR TITLE
add LSTM model

### PR DESCRIPTION
**Pull request by AgaMiko**
Not the greatest model but can be used to compare performance with transformers. In general, LSTM should be worse.

### CODE REVIEW

File: models/model_loader.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class ModelLoader:
         "fake_transformer": transformers.FakeTransformer,
         "hubert_transformer": transformers.HubertTransformer,
         "sign_language_transformer": transformers.SignLanguageTransformer,
+        "lstm": transformers.LSTM,
```
Comment (majsylw) [inline models/model_loader.py:20]: is it a totaly new part of model or just replacement for "sign_language_transformer"?

Comment (AgaMiko) [reply models/model_loader.py:20]: Can be used instead of sign_language_transformer.  It's a simpler version of the "transformer" model (it is not a transformer though). Good for comparison. If the results for LSTM and transformers are the same, it might mean that it is still very difficult to train the models with used problem definition, loss functions, or data.

**Review by MilenaOlech — APPROVED** [review]

File: models/transformers/lstm.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import torch
+class LSTM(torch.nn.Module):
+    """Basic LSTM model"""
+
+    def __init__(self,
```
Comment (wiktorfilipiuk) [inline models/transformers/lstm.py:5]: To keep consistency with other models please use new constructor's parameters format like so:
def __init__(
        self,
        feature_extractor_parameters: Dict = None,
        transformer_parameters: Dict = None,
        train_parameters: Dict = None
    ):

File: models/transformers/lstm.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            output_size (int): Output size. Should be equal to the selected representation size. 
+            num_frames (int): Number of input frames.
+            feedforward_size (int): Hidden size of an LSTM model. Defaults to 512.
+            num_encoder_layers (int): Number of LSTM layers. Defaults to 2.
+            dropout_rate (float): Dropout stats. Should be between 0 and 1. Deafults to 0.2.
+            
+        """
+        super().__init__()
+        self.LSTM = torch.nn.LSTM(input_size=input_size,
+                            hidden_size=feedforward_size,
```
Comment (wiktorfilipiuk) [inline models/transformers/lstm.py:26]: missing indent

**Review by adrianlachowicz — APPROVED** [review]

**Review by aga0 — APPROVED** [review]
LGTM

File: models/transformers/lstm.py (hunk 2 above)
Comment (AgaMiko) [reply models/transformers/lstm.py]: Updated

File: models/transformers/lstm.py (hunk 3 above)
Comment (AgaMiko) [reply models/transformers/lstm.py]: Updated
